dotnet/Cucumber.Messages/NdjsonSerializer.cs

```diff
@@ -3,23 +3,30 @@
 
 namespace Cucumber.Messages
 {
-    public class NdjsonSerializer
+    public static class NdjsonSerializer
     {
+        private static object lockObject = new object();
```

Comment:
This solution is probably good enough, but for such thread-safe initializations, I generally prefer using the `Lazy<T>` class, because the lock object version needs to aquire a lock for every single property get, even after the initialization period.

Reply:
Ah. I had completely forgotten about the option to use Lazy. (I'm too old school for such new sparkly things) 
I'll change it and resubmit.

Reply:
Done. Revised the PR to make this use Lazy<T>